src/ocamlorg_frontend/components/learn_tutorial_block.eml

```diff
@@ -0,0 +1,53 @@
+let tutorial_block_content 
+~title
+~heading
+~description
+~(tutorial_links:string list)
+~see_more
+= 
+    <div class="w-full md:w-80">
+        <div class="mt-4 text-white text-sm text-left font-medium leading-7 tracking-widest"> <%s title %> </div>
+        <h2 class="mt-2 text-white text-2xl text-left font-normal leading-7"> <%s heading %> </h2>
+        <div class="mt-4 h-16 text-white text-left text-md font-normal leading-7"> <%s description %> </div>
```

Comment:
`h-16`... what happens when someone puts text in this that makes it overflow? Should this be cut off? Should it be an ellipsis? Or should the height just give way and grow beyond this?

What it does now: It will overflow into the content below.